core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -399,6 +406,16 @@ object BootMonitors extends LazyLogging {
       )
     }
 
+  def startWorkspaceResourceMonitor(
+    system: ActorSystem,
+    dataSource: SlickDataSource,
+    workspaceManagerDAO: WorkspaceManagerDAO,
+    samDAO: SamDAO
+  ) = {
+    val jobRunners = List()
+    system.actorOf(WorkspaceResourceMonitor.props(dataSource, jobRunners))
```

Comment:
Should this monitor get its own dispatcher?

Reply:
If by dispatcher, you mean actor system, then no; I don't think so:
1. the Akka docs recommend one actor system per application, since an actor system itself is pretty heavy weight 
2. we're already running this in the top-level of the hierarchy, so I don't think we really get anything for it